beacon.el

```diff
@@ -246,6 +246,10 @@ COLORS applied to each one."
   ;; The after-string must not be longer than the remaining columns
   ;; from point to right window-end else it will be wrapped around.
   (let ((colors (seq-take colors (- (window-width) (beacon--visual-current-column) 1))))
+    (if (and (derived-mode-p 'org-mode)
+             (org-invisible-p))
+        (progn (goto-char (org-find-visible))
```

Comment:
Not sure how I feel about beacon moving the point, seems like it could lead to some unexpected corner-cases.
Maybe with a save-excursion....